docs/content/latest/architecture/core-functions/universe-creation.md

```diff
@@ -30,9 +30,8 @@ their load, but no tablets would exist in the system yet.
 ## An example
 
 Let us illustrate this with our usual example of creating a 4-node YugabyteDB universe with a
-replication factor of 3. In order to do so, first the three masters are started in the create mode
-instructing them to that this is a brand new universe create. This is done explicitly to prevent
-accidental errors in creating a universe while it is already running.
+replication factor of 3. First the three masters are started in create mode. This is done explicitly
```

Comment:
Hmm...I need to look closer at this. The only place I know we have "create" is in `yb-ctl` and I wouldn't call it a mode. And, this is in the Architecture section...sounds wrong here. But you're simplifying and shortening this, so go ahead with this change and I'll review this topic later (soon).